CONTRIBUTORS.md

```diff
@@ -7,7 +7,7 @@ without your contributions. Thank you! 💖
 New non-code contributors should add their name to the bottom of this list on their first
 community contribution, like writing a blog, giving a talk, or organizing a meetup. 
 Please either submit a pull request adding your name, or mention it to another organization member
-and we will add you. **All** contributors belong here. 💯
+and we will add you. Use the commit message to briefly describe or link to your work. *All** contributors belong here. 💯
```

Comment:
```suggestion
and we will add you. Use the commit message or PR description to briefly describe or link to your work. *All** contributors belong here. 💯
```

Reply:
For some of the people with non-code contributions, they may not know how to add a commit message so I think they should also be able to add it to the PR description.

Reply:
Sounds good!